client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -71,16 +83,73 @@ const AccountBalancesTabPanel: React.FC = () => {
 			{ ( tab: BalanceTab ) => (
 				<Flex gap={ 0 } className="wcpay-account-balances__balances">
 					<BalanceBlock
-						type="available"
+						id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
+						title={ fundLabelStrings.available }
 						amount={ tab.availableFunds }
 						currencyCode={ tab.currencyCode }
 						isLoading={ isLoading }
+						tooltip={
+							<BalanceTooltip
+								label={ `${ fundLabelStrings.available } tooltip` }
+								content={
+									tab.availableFunds < 0 ? (
+										<a
+											target="_blank"
+											rel="noopener noreferrer"
+											href={
+												documentationUrls.negativeBalance
+											}
+										>
+											{
+												fundTooltipStrings.availableNegativeBalance
+											}
+										</a>
+									) : (
+										<>
+											{ fundTooltipStrings.available }{ ' ' }
+											<a
+												target="_blank"
+												rel="noopener noreferrer"
+												href={
+													documentationUrls.depositSchedule
+												}
+											>
+												{ learnMoreString }
+											</a>
+										</>
+									)
+								}
+							/>
+						}
 					/>
 					<BalanceBlock
-						type="pending"
+						id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
+						title={ fundLabelStrings.pending }
 						amount={ tab.pendingFunds }
 						currencyCode={ tab.currencyCode }
 						isLoading={ isLoading }
+						tooltip={
+							<BalanceTooltip
+								label={ `${ fundLabelStrings.pending } tooltip` }
+								content={
+									<>
+										{ sprintf(
```

Comment:
I have similar situation with [my PR](https://github.com/Automattic/woocommerce-payments/pull/5861/files#r1151093049).

I agree that it's nice to have strings in one place, in `strings.ts`, but here we have dynamic string and we assume that the string has a delay days parameter.

I don't know the ideal solution though. I consider my concern as `nit`, so feel free to ignore.

Reply:
I don't think we should factor strings out of the components – it decouples the format string (placeholders / interpolation) from the replacements. 

@Jinksi do you know why we have separate strings files? What do you recommend?

Reply:
We decided to follow this `strings.ts` convention when we were discussing how we'll setup the skeleton components. It seems to be used throughout WCPay.

There are pros/cons, I don't have a preference either way.

I've moved these strings inside the component file in 10bc97e8a0a2d20518f106dc759fa38e1906c54c; it becomes less cumbersome when dealing with `interpolateComponents`.

Reply:
> We decided to follow this strings.ts convention when we were discussing how we'll setup the skeleton components. It seems to be used throughout WCPay.

TIL, good to know. Something for the front-end guild to consider maybe.